dpypelines/pipeline/dataset_ingress_v1.py

```
@@ -200,3 +268,124 @@ def dataset_ingress_v1(files_dir: str, pipeline_config: dict):
     )
 
     print("Worked. I ran to completion.")
+
+    # Upload output files to Upload Service
+    try:
+        # Get Upload Service URL from environment variable
+        upload_url = os.environ.get("UPLOAD_SERVICE_URL", None)
```

can you put the env var checks at the top of the script please (all of them please). Better to fail early, some of the transforms might take a while.